unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/BackpackAnalyticsController.cs

```diff
@@ -0,0 +1,56 @@
+﻿using DCLServices.WearablesCatalogService;
+using System.Collections.Generic;
+
+namespace DCL.Backpack
+{
+    public class BackpackAnalyticsController
+    {
+        private const string EQUIP_WEARABLE_METRIC = "equip_wearable";
+
+        private readonly IAnalytics analytics;
+        private readonly INewUserExperienceAnalytics newUserExperienceAnalytics;
+        private readonly IWearablesCatalogService wearablesCatalogService;
+
+        public BackpackAnalyticsController(
+            IAnalytics analytics,
+            INewUserExperienceAnalytics newUserExperienceAnalytics,
+            IWearablesCatalogService wearablesCatalogService)
+        {
+            this.analytics = analytics;
+            this.newUserExperienceAnalytics = newUserExperienceAnalytics;
+            this.wearablesCatalogService = wearablesCatalogService;
+        }
+
+        public void SendNewEquippedWearablesAnalytics(List<string> oldWearables, List<string> newWearables)
+        {
+            foreach (string newWearable in newWearables)
```

Comment:
I did it in this way because is the way how is done in our current backpack and I prefer to keep as it is in order to avoid to change how we are registering this specific metric.

Reply:
👍